### DISCUSSION TITLE
Partials override to move "Last Updated" to top of page?

### DISCUSSION THREAD

**Discussion by aforsythe** [squidfunk/mkdocs-material#5846] [Q&A] [answered]
Sorry for the basic question.  I'm just banging my head against the wall trying to figure out what I'm doing wrong. 

I'm using `git-revision-date-localized` and I would like to move the "Last Updated" notification to the top of the page (preferable under the title, but I haven't gotten that far yet.

I've added a `custom_dir: mkdocs/overrides` to my mkdocs.yml

I've created a `overrides/partials/main.html`

The contents are 

```
{% extends "base.html" %} 
{% block content %}
{{ page.meta.revision_date }}
{{ super() }}
{% endblock content %}
```

This seems to have no effect.

Thanks for any help in advance.

**Comment by squidfunk** [comment]
From the information you gave us, it looks like [you put `main.html` in the wrong place](https://squidfunk.github.io/mkdocs-material/customization/#overriding-blocks). It should be:

``` sh
.
├─ overrides/
│  └─ main.html
└─ mkdocs.yml
```

**Reply by aforsythe** [reply]
I've moved `main.html` to `overrides` rather than `partials`, but still no luck.

My structure looks like this now.

```
├─latex/
├─other stuff/
├─mkdocs/
│  └─ docs/
│  └─ overrides/
│          └─ main.html
└─ mkdocs.yml
└─ Dockerfile
```

I tired moving `overrides` up to to the root next to the `mkdocs.yml` and adjusted my `custom_dir:` accordingly, but no luck that way either.

**Reply by aforsythe** [reply]
Here's the relevant sections of the `mkdocs.yml`
```
site_dir: /site
docs_dir: mkdocs/docs
...
theme:
  name: 'material'
  custom_dir: mkdocs/overrides
```

**Reply by squidfunk** [reply]
If you can provide a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/), we can look into it. We need something we can run.

**Reply by aforsythe** [reply]
Thanks .. here's a minimal reproduction.

[exmaple.zip](https://github.com/squidfunk/mkdocs-material/files/12373332/exmaple.zip)

**Reply by squidfunk** [reply]
Thanks for providing the reproduction. You're deviating from the recommended file structure, as you're putting stuff into a subfolder. I've tested, and it does not work with other themes than Material for MkDocs as well, so it's a general MkDocs problem. Your options:

- Use the [recommended file structure](https://squidfunk.github.io/mkdocs-material/creating-your-site/)
- Raise a question (or issue) to the maintainers of MkDocs. You might have hit a bug, but I'm not sure.

**Comment by aforsythe** [comment]
Thanks but modifying the structure to the recommended file structure doesn't seem to fix it.

Further, modifying `main.html` shows that the override is being picked up even with the subdirectory structure.

modified `main.html`
```
{% extends "base.html" %} 
{% block htmltitle %}
  <title>Lorem ipsum dolor sit amet</title>
{% endblock %}
{% block content %}
{{ page.meta.revision_date }}
{{ super() }}
{% endblock content %}
```

Here's a version with the modified file structure showing that the page revision date isn't being moved to the top even when following the recommended file structure.
[exmaple_revised_structure.zip](https://github.com/squidfunk/mkdocs-material/files/12389027/exmaple_revised_structure.zip)

Could this be an issue with `git-revision-date-localized`?

**Comment by aforsythe** [comment, marked as answer]
Well this is slightly embarrassing. 

Turns out I was using the wrong variable name for `git-revision-date-localized`.  Should be `page.meta.git_revision_date_localized` instead of `page.meta.revision_date`.  

I'm still trying to figure out how to disable the default placement of "Last updated" when using the variable and how I can place the variable below the page title, but that seems like a question for @timvink.  

By the way, this works with the subdirectory structure I was originally using.

**Reply by timvink** [reply]
Hi there! I didn't read your full thread, but see the documentation of the `git-revision-date-localized` plugin on overriding themes: https://timvink.github.io/mkdocs-git-revision-date-localized-plugin/howto/override-a-theme/

The highlighted bit is what inserts "Last updated" (based on the locale used):

<img width="649" alt="image" src="https://github.com/squidfunk/mkdocs-material/assets/5570380/c77c8078-f40a-41c9-b9c4-1838a61e05ab">
